spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,17 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
```

Comment:
I think you could remove the comments from this file

Reply:
I kept them to find the links easier, but I think it's pretty standard anyway. Done!